packages/wrangler/src/index.ts

```diff
@@ -1129,6 +1205,18 @@ export async function main(argv: string[]): Promise<void> {
 			await captureGlobalException(e);
 		}
 
+		const durationMs = Date.now() - startTime;
+
+		void dispatcher?.sendNewEvent("wrangler command errored", {
```

Comment:
Nit: It looks like neither `sendNewEvent` nor `sendEvent` actually await for anything. We can make them sync and then there is no need for `void` here.

Reply:
ed450154f8232013f0fba2976b824db01af6d821